src/app/common/manager/message_manager.ts

```diff
@@ -19,6 +19,7 @@ export async function searchMessageById(guild: $TSFixMe, channelId: $TSFixMe, me
             // fetch(mid)とすれば、cache見てなければフェッチしてくる
             message = await channel.messages.fetch(messageId);
         } catch (error) {
+            message = null;
```

Comment:
catchからreturnが近いからこれでもいいけど、厳密にはここでreturnしたほうがいいけど、好みかな
```suggestion
            logger.warn('message missing');
            return null;
```

Reply:
ここはfetch失敗すると直のAPIエラーになる点を強調しときたいのでそのままにしときます。
fetch部分自体コードととしてはあんまり良くないとは思うのでどうにかしたい。